packages/daimo-userop/src/daimoOpSender.ts

```diff
@@ -0,0 +1,275 @@
+import { derKeytoContractFriendlyKey } from "@daimo/common";
+import * as Contracts from "@daimo/contract";
+import { Client, ISendUserOperationOpts } from "userop";
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
```

Comment:
can this be just `from "userop"`? fine anyway since #264 removes all this code anyway

Reply:
not exported directly. i changed it to `import type` to show its just a type annotation